### PR TITLE
OBSDOCS-185: changes-to-sample-code-for-repeat-interval-settings

### PR DESCRIPTION
**Pull request by bburt-rh**
Summary: This PR adjusts the description of the `group_interval` setting and to the sample code for the `repeat_interval` settings.

- Aligned team: DevTools
- For branches: 4.10+
- Jira: https://issues.redhat.com/browse/OBSDOCS-185
- Direct link to doc preview: https://61754--docspreview.netlify.app/openshift-enterprise/latest/monitoring/managing-alerts.html#applying-custom-alertmanager-configuration_managing-alerts
- SME review: @simonpasquier (done) 
- QE review: @juzhao (done)
- Peer review: @gwynnemonahan (done)

### CODE REVIEW

File: modules/monitoring-applying-custom-alertmanager-configuration.adoc [hunk 1]
```diff
@@ -57,7 +57,7 @@ The default value is 30 seconds (`30s`).
 <2> The `group_interval` value specifies how much time must elapse before Alertmanager sends a notification about new alerts added to a group of alerts for which an initial notification was already sent.
 The default is five minutes (`5m`).
 <3> The `repeat_interval` value specifies the minimum amount of time that must pass before an alert notification is repeated.
-The default is four hours (`4h`).
+The default setting for the Cluster Monitoring Operator is 12 hours (`12h`).
```
Comment (simonpasquier) [inline modules/monitoring-applying-custom-alertmanager-configuration.adoc:60]: 4h is the default value in Alertmanager. CMO today will provision a default Alertmanager configuration which happens to have a 12h value but I wouldn't mention it in the docs since it could change in the future and existing clusters wouldn't see the new default.

TL;DR: I'd remove the default values for all interval  parameters.

File: modules/monitoring-applying-custom-alertmanager-configuration.adoc [hunk 2]
```diff
@@ -104,6 +104,7 @@ receivers:
 ----
 +
 With this configuration, alerts of `critical` severity that are fired by the `example-app` service are sent using the `team-frontend-page` receiver. Typically these types of alerts would be paged to an individual or a critical response team.
+Also, because the `repeat_interval` value is set to less than the `group_interval` value, notifications will be repeated at each group interval.
```
Comment (simonpasquier) [inline modules/monitoring-applying-custom-alertmanager-configuration.adoc:107]: this only applies for the Watchdog alert. I'm a bit torn on mentioning such level of details...

Comment (danielmellado) [reply modules/monitoring-applying-custom-alertmanager-configuration.adoc:107]: +1, if these are user oriented docs, I think this is getting too much into specific implementation details.

Comment (bburt-rh) [reply modules/monitoring-applying-custom-alertmanager-configuration.adoc:107]: Thanks, @simonpasquier and @danielmellado - can you ptal at the comments in the Jira issue and add your thoughts about how best to deal with the issue as described by the support engineer? https://issues.redhat.com/browse/RHDEVDOCS-185

File: modules/monitoring-applying-custom-alertmanager-configuration.adoc (hunk 1 above)
Comment (bburt-rh) [reply modules/monitoring-applying-custom-alertmanager-configuration.adoc:60]: I removed all default value information.

File: modules/monitoring-applying-custom-alertmanager-configuration.adoc (hunk 2 above)
Comment (bburt-rh) [reply modules/monitoring-applying-custom-alertmanager-configuration.adoc]: I removed this line from the PR. Are you both good with having `repeat_interval` set to 2m in the sample?

**Comment by juzhao** [comment]
LGTM, waiting for others to review

**Review by simonpasquier — COMMENTED** [review]
/lgtm

**Review by gwynnemonahan — APPROVED** [review]
Rendering issues resolved. LGTM!
